### PR TITLE
Strimzi 0.7.0 release blog post

### PR DESCRIPTION
**Pull request by ppatierno**
This PR is about a blog post on 0.7.0 release.
Because we are late on that and we are closed to the new 0.8.0 release, I focused on the two main features (exposing Kafka outside and SCRAM SHA auth) but just referring to the others.
I don't think we have to spend so much time on it, I'd like to publish it tomorrow morning.
Anyway, any grammar or content feedback is welcome!

### CODE REVIEW

File: _posts/2018-10-01-strimzi-0.7.0-release.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+# Exposing Kafka cluster to the outside
+
+The Strimzi community has been asking for it for long time and finally it's here!
+The Kafka cluster is now accessible from the outside of the Kubernets/OpenShift cluster where it's deployed.
+Your application doesn't need to be containarized and running in the same Kubernetes/OpenShift cluster for connecting to the Kafka cluster; your application can be any client running outside of it.
+
+There are three different ways for doing that using:
+
+* OpenShift routes (so OpenShift only)
```
Comment (scholzj) [inline _posts/2018-10-01-strimzi-0.7.0-release.md]: I would probably use capital R ... Routes ... to match LoadBalancers and NodePorts.

File: _posts/2018-10-01-strimzi-0.7.0-release.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+
+We are delighted to announce the new Strimzi 0.7.0 release with many awesome new features!
+
+<!--more-->
+
+# Exposing Kafka cluster to the outside
+
+The Strimzi community has been asking for it for long time and finally it's here!
+The Kafka cluster is now accessible from the outside of the Kubernets/OpenShift cluster where it's deployed.
+Your application doesn't need to be containarized and running in the same Kubernetes/OpenShift cluster for connecting to the Kafka cluster; your application can be any client running outside of it.
```
Comment (tombentley) [inline _posts/2018-10-01-strimzi-0.7.0-release.md]: You client application

File: _posts/2018-10-01-strimzi-0.7.0-release.md [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+
+There are three different ways for doing that using:
```
Comment (tombentley) [inline _posts/2018-10-01-strimzi-0.7.0-release.md]: There are three different way to expose the Kafka cluster: 

* Using OpenShift `Routes`
* Using `LoadBalancers`
* Using `NodePorts`

File: _posts/2018-10-01-strimzi-0.7.0-release.md [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+
+* OpenShift Routes (so OpenShift only)
+* LoadBalancers
+* NodePorts
+
+In order to do that, the new `external` listener was added to the `Kafka` resource.
```
Comment (tombentley) [inline _posts/2018-10-01-strimzi-0.7.0-release.md]: The new `Kafka.spec.kafka.listeners.external` listener configures whether and how the Kafka cluster is exposed.

File: _posts/2018-10-01-strimzi-0.7.0-release.md [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+It can be of three different types : `route`, `loadbalancer` and `nodeport`.
+
+Here's an example snippet about exposing the Kafka cluster using OpenShift routes:
```
Comment (tombentley) [inline _posts/2018-10-01-strimzi-0.7.0-release.md]: Here's an example snippet that exposes the Kafka cluster using OpenShift routes:

File: _posts/2018-10-01-strimzi-0.7.0-release.md [hunk 6]
```diff
@@ -0,0 +1,151 @@
...
+        type: route
+    ...
+```
+
+When the Kafka cluster is exposed by using OpenShift routes, a dedicated `Route` is created for every Kafka broker pod.
+An additional `Route` is created to serve as a Kafka bootstrap address.
+Kafka clients can use these `Routes` to connect to Kafka on port 443.
+
+When the Kafka cluster is exposed by using LoadBalancers, a new `Service` of type `LoadBalancer` is created for every Kafka broker pod.
+An additional loadbalancer is created to serve as a Kafka bootstrap address.
```
Comment (tombentley) [inline _posts/2018-10-01-strimzi-0.7.0-release.md]: An additional `LoadBalancer`...

File: _posts/2018-10-01-strimzi-0.7.0-release.md [hunk 7]
```diff
@@ -0,0 +1,151 @@
...
+Loadbalancers listen to connections on port 9094.
+
+When the Kafka cluster is exposed by using NodePorts, `Services` of type `NodePort` are created.
+Kafka clients connect directly to the OpenShift or Kubernetes cluster nodes.
+You must enable access to the ports on these nodes for each client (for example, in firewalls or security groups).
+Each Kafka broker pod is then accessible on a separate port.
+Additional `NodePort` type `Service` is created to serve as a Kafka bootstrap address.
+
+It's important to highlight that the connection from outside is possible only through TLS.
```
Comment (tombentley) [inline _posts/2018-10-01-strimzi-0.7.0-release.md]: is currently only possible through `TLS`.

File: _posts/2018-10-01-strimzi-0.7.0-release.md [hunk 8]
```diff
@@ -0,0 +1,151 @@
...
+
+Of course, as the other listeners, you can configure the type of `authentication` you want for the `external` listener from the currently supported: TLS and SASL SCRAM-SHA 512 (new with 0.7.0 release).
```
Comment (tombentley) [inline _posts/2018-10-01-strimzi-0.7.0-release.md]: ...as with the other listeners...

**Pull request merged**